boards/arm/phy62xx/phy6222/scripts/Make.defs

```diff
@@ -53,9 +53,6 @@ NXFLATLDFLAGS1 = -r -Wl,-d -Wl,-warn-common
 NXFLATLDFLAGS2 = $(NXFLATLDFLAGS1) -T$(TOPDIR)/binfmt/libnxflat/gnu-nxflat-pcrel.ld -Wl,-no-check-sections
 LDNXFLATFLAGS = -e main -s 2048
 
-ifneq ($(CROSSDEV),arm-nuttx-elf-)
```

Comment:
can we merge this change separately ? The CI is failing now on the mainline

Reply:
Or this is unrelated to CI failure?

Reply:
New gcc can only recognize --nostartfile instead of -nostartfiles bellow (they are already deleted from other socs)

Reply:
I'll submit it separately soon.

Reply:
There is
```
ifneq ($(CROSSDEV),sparc-nuttx-elf-)
  LDFLAGS += -nostartfiles -nodefaultlibs
endif
```
that most probably also should be deleted. But let's fix CI first